diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/WhiteSpaceRule.kt

```diff
@@ -198,10 +208,11 @@ class WhiteSpaceRule(private val configRules: List<RulesConfig>) : Rule("horizon
 
     private fun handleBinaryOperator(node: ASTNode) {
         val operatorNode = if (node.elementType == OPERATION_REFERENCE) node.firstChildNode else node
-
-        if (node.elementType == OPERATION_REFERENCE && node.treeParent.elementType.let { it == BINARY_EXPRESSION || it == POSTFIX_EXPRESSION } ||
+        if (node.elementType == EQ && node.treeParent.elementType == OPERATION_REFERENCE) return
```

Comment:
What does this case correspond to?

Reply:
Because `OPERATION_REFERENCE ` sometime contains `EQ`. I had test for this.

Reply:
Yes, and what is an example in code? And why in this case you `return` from the method?

Reply:
Because I check this in the `handleBinaryOperator`

Reply:
Right, I see now.